pages/table/editable.page.tsx

```diff
@@ -207,7 +207,7 @@ const Demo = forwardRef(
       let value = newValue;
       await new Promise(r => setTimeout(r, 1000));
       errorsMeta.delete(currentItem);
-      if (value === 'inline') {
+      if (typeof value === 'string' && value.includes('inline')) {
         errorsMeta.set(currentItem, 'Server does not accept this value, try another');
         throw new Error('Inline error');
```

Comment:
Should we reflect the server error in the demo? The focusing works but there is no error state/text visible.

Reply:
It already displays the server side error "Server does not accept this value, try another" (if there isn't another client-side error present already)